weatherApp.js

```diff
@@ -192,26 +196,59 @@ const handleDayClick = (day, divDay) => {
 }
 
 const createHourBoxes = (day) => {
-	const hourAndTempList = day.map(el => {
+	const hourAndTempList = day.map((el, index) => {
 		const hours = el.dt_txt.split(" ")[1].split(":", 2);
 		const temp = Math.round(el.main.temp - KELWIN_DIFF);
 		const hourAndTemp = {hour: `${hours[0]}:${hours[1]}`, celsius: temp};
 		
 		const properIcon = getWeatherDayIcon([el]);
 		
+		const accordionHeadline = upperFirstLetter(el.weather[0].description)
+		const windSpeed = Math.round(el.wind.speed*(METERS_TO_KILOMETER/SECONDS_TO_HOUR));
+		const pressure = Math.round(el.main.pressure);
 		const divHourBox = document.createElement("div");
-		
 		divHourBox.innerHTML = `
-			<span class="hour">${hourAndTemp.hour}</span>
-			<span class="meteoicon" data-icon=${properIcon.weatherIcon}></span>
-			<span class="hour-temp">${hourAndTemp.celsius} °C</span>
-		`
+			<label class="hour-tab" for="h-${index}">
+				<span class="hour">${hourAndTemp.hour}</span>
+				<span class="meteoicon" data-icon=${properIcon.weatherIcon}></span>
+				<span class="hour-temp">${hourAndTemp.celsius} °C</span>
+			</label>
+			<div class="in-hour-tab-accordion">
+				<span class="accordion-head">${accordionHeadline}</span>
+				<div class="accordion-pressure">
+					<div class="accordion-tag">
+						<span class="accordion-label">Pressure </span>
+						<span class="accordion-data">${pressure} hPa</span>
+					</div>
+				</div>
+				<div class="accordion-humidity">
+					<div class="accordion-tag">
+						<span class="accordion-label">Humidity </span>
+						<span class="accordion-data">${el.main.humidity } %</span>
+					</div>
+				</div>
+				<div class="accordion-wind-speed">
+					<div class="accordion-tag">
+						<span class="accordion-label">Wind speed </span>
+						<span class="accordion-data">${windSpeed} km/h</span>
+					</div>
+				</div>				
+			</div>
+		`;
+		/*createAccordionContent(day, divHourBox);*/
+		divHourBox.tabIndex = "1";
 		divHourBox.className = "hour-box";
 		return divHourBox;
 	})
 	return hourAndTempList;
 }
 
+/*const createAccordionContent = (day, parentHourDiv) => {
+	console.log("day", day);
+	const accordionDiv = `dupadupadpa`
+	return parentHourDiv.insertAdjacentHTML('beforeend',`<div>${accordionDiv}</div>`)
+}*/
```

Comment:
Unnecessary comment

Reply:
Ok, I had doubts if it wouldn't be better to use another function for creating accordion content.